Processing/BRCA_Deep_Dive.Rmd

```diff
@@ -530,52 +468,102 @@ well_data <- read_csv('data/park/BRCA/manual/BRCA_Well_Data.csv', na = c("NaN",
   arrange(date) %>%
   dplyr::mutate(static_ft_c = static_ft) %>%
   group_by(well) %>%
-  mutate(static_ft_c = na.approx(static_ft, na.rm = FALSE)) %>%
-  dplyr::mutate(aq_volume_ft3 = 
-                  map_dbl(static_ft_c, ~aquifer_volume(dtw_ft = .)))
- 
+  mutate(static_ft_c = na.approx(static_ft, na.rm = FALSE))
+
+# Summarize to annual mean 
 well_data_an <- well_data %>%
   dplyr::mutate(year = year(date)) %>%
   dplyr::group_by(year, well) %>%
   dplyr::summarize(static_ft_c = mean(static_ft_c)) %>%
   dplyr::mutate(aq_volume_ft3 = 
-                  map_dbl(static_ft_c, ~aquifer_volume(dtw_ft = ., full_thickness_ft = 30)))
+                  map_dbl(static_ft_c, 
+                          ~aquifer_volume(dtw_ft = ., full_thickness_ft = 30)))
```

Comment:
Confirmed - well_data_an is a variable that wasn't used. It can actually be deleted as a whole but I'm going to keep it for now. The aquifer_volume part was added on during testing and is not necessary.